toqito/channel_ops/complementary_channel.py

```diff
@@ -0,0 +1,45 @@
+"""Computes the complementary channel/map of a superoperator."""
+
+import numpy as np
+
+
+def complementary_channel(kraus_ops: list[np.ndarray]) -> list[np.ndarray]:
+    r"""Compute the Kraus operators for the complementary map of a quantum channel.
+
+    (Section: Representations and Characterizations of Channels from :cite:`Watrous_2018_TQI`).
+
+    The complementary map is derived from the given quantum channel's Kraus operators by
+    rearranging the rows of the input Kraus operators into the Kraus operators of the
+    complementary map.
+
+    Specifically, for each Kraus operator :math:`K_i` in the input channel :math:`\Phi`,
+    we define the complementary Kraus operators :math:`K_i^C` by stacking the rows of
+    :math:`K_i` from all Kraus operators vertically.
+
```

Comment:
this docstring is missing an example

Reply:
Ahh yes, I am adding the docstring here as well.